base/deprecated.jl

```diff
@@ -1821,6 +1821,8 @@ function toc()
     return t
 end
 
+@deprecate sparse(s::UniformScaling, m::Integer) sparse(s, m, m)
```

Comment:
Probably just need an `import SparseArrays.sparse` here

Reply:
Yes, something of that kind should do the trick. I went with `@eval SparseArrays @deprecate ...`; let's see what CI says. Thanks! :)